### DISCUSSION TITLE
Supported license file format

### DISCUSSION THREAD

**Discussion by gee4vee** [microsoft/vscode-discussions#412] [Extension Development QnA] [answered]
We need to include multiple translations of our license in our license file. What file formats does the marketplace support to ensure the characters can be rendered?

**Comment by isidorn** [comment]
Duplicate of https://github.com/microsoft/vscode-discussions/discussions/399

**Comment by gee4vee** [comment]
My question is different @isidorn. I'm asking here what file type is supported by the marketplace (txt, rtf, html, etc.).

**Reply by isidorn** [reply]
I am not sure. @seaniyer should be able to help here

**Comment by seaniyer** [comment, marked as answer]
@gee4vee, Just text format.
